.gitignore

```diff
@@ -0,0 +1,2 @@
+node_modules
```

Comment:
`.gitignore` should be inside of your project directory, it doesn't need to be in the repository root.